### PR TITLE
Add Diggie to the partners page

### PR DESCRIPTION
**Pull request by lopburny**
### Description
Hi folks, Thank you for all the work. 
Adding Diggie logo and link to the partners page. 

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by stockholmux** [comment]
Thanks for the PR @lopburny! 


I see the logo and the link go to Diggie in this PR.Usually the partners pages link to your company page and uses your companies logo. 

If you want to feature your project, you can add it to the [Community Projects page](https://opensearch.org/community_projects).

What do you think?

**Comment by lopburny** [comment]
Updated our company logo and link, also moved Diggie to the community projects page! 
One thing, we're currently working on the license text, is it OK to list the projects with a blank license link first, and update it in a few weeks?

File: _community_projects/diggie.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Diggie
+description: Professional GUI client for AWS OpenSearch, desktop app of Mac / Windows that makes operating with OpenSearch easier. This project is currently a developer preview version. Feedbacks and opinions are welcome.
+owner: Serverless Operations, Inc.
+owner_link: https://serverless.co.jp
+link: https://github.com/DiggieApp/Diggie
+download_link: https://github.com/DiggieApp/Diggie/releases/tag/0.1.0
+license: Diggie EULA (developer preview)
+license_link: "#" # in preparation
```
Comment (stockholmux) [inline _community_projects/diggie.md:9]: I'd rather this lead to a 404 than to `#`. If you know the URL where your license will be already, then just put it in now and we don't have to change it later.

**Review by stockholmux — CHANGES_REQUESTED** [review]

File: _community_projects/diggie.md (hunk 1 above)
Comment (lopburny) [reply _community_projects/diggie.md]: @stockholmux I get it 🙌  Added license_link and we'll make the link available soon!

**Review by stockholmux — APPROVED** [review]
LGTM!

**Pull request merged**